staging/src/k8s.io/client-go/kubernetes/timeout_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package kubernetes_test
```

Comment:
We could leave it in the _test pkg for isolation purposes while still moving the location of the files.

Reply:
sorry, i dont quit understand you?

Reply:
For client-go was widely imported by other projects, people will frequently be
redirected to the client-go source package, and they will take a quick look
at the source folder. `kubernetes_test` would make noise for them to
deep further (i am one of the victim but not the last).

Reply:
Why would this be noise? It's a separate package.

from `go help test`, see also https://golang.org/cmd/go/#hdr-Test_packages
```
Test files that declare a package with the suffix "_test" will be compiled as a separate package, and then linked and run with the main test binary. 
```

Reply:
I may be confused

Reply:
sorry for the confusion, the main purpose of this pr is to remove the **directory** kubernetes_test